spec/iot-js-spec.md

```diff
@@ -123,31 +118,29 @@ dictionary OicDeviceInfo {
 interface OicClient: EventTarget {
   // client API: discovery; Promise succeeds when request is successfully sent
   Promise<void> findResources(OicDiscoveryOptions options);  // GET oic/res
-  Promise<void> findDevices(OicDiscoveryOptions options);  // GET /oic/d
+  Promise<void> findDevices();  // GET /oic/d
 
   // client API: CRUDN
-  Promise<OicResource> createResource(OicResourceInit resource);  // create remote res
-  Promise<OicResource> retrieveResource(USVString resourceId);  // get remote res
-  Promise<void> updateResource(USVString resourceId, OicResourceInit resource);  // post or put remote res
-  Promise<void> deleteResource(USVString resourceId);  // delete remote res
-  Promise<OicResource> startObserving(USVString resourceId);  // retrieve + observe flag
-  Promise<void> cancelObserving(USVString id);
-
-  attribute EventHandler<OicResourceChangeEvent> onresourcechange;  // observe
```

Comment:
I would like to rename from onresourcechange to onresourceupdate, since that is more in line with the Core spec.
And no, we don't need startObserve() and cancelObserve(), since we can hook to the EventHandler anyway. It was my first uneducated attempt to support observe, let's not make a legacy out of it :).